docs/release-notes/8.0/8.0.35-30.0.md

```diff
@@ -2,6 +2,18 @@
 
 Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
 
+## Supported versions
+
+Percona XtraBackup 8.0.35 allows taking backups of MySQL-compatible databases in the following versions:
+
+* Percona Server for MySQL 8.0.34, and MySQL 8.0.34
+
+* Percona Server for MySQL 8.0.35, and MySQL 8.0.35
+
+* Percona Server for MySQL 8.0.36 and MySQL 8.0.36
+
```

Comment:
In the release notes - just add a link to the Supported versions.

Reply:
done